lib/mayaUsd/fileio/shading/symmetricShaderWriter.h

```diff
@@ -53,14 +53,14 @@ PXR_NAMESPACE_OPEN_SCOPE
 /// \code
 /// TF_REGISTRY_FUNCTION(UsdMayaShaderWriterRegistry)
 /// {
-///     PxrUsdTranslators_SymmetricShaderWriter::RegisterWriter(
+///     UsdMayaSymmetricShaderWriter::RegisterWriter(
 ///         TfToken("myMayaNodeTypeName"),
 ///         TfToken("MyUsdShaderId"),
 ///         /* materialConversionName = */ TfToken("myMaterialConversionName"));
 /// };
 /// \endcode
 ///
-class PxrUsdTranslators_SymmetricShaderWriter : public UsdMayaShaderWriter
+class UsdMayaSymmetricShaderWriter : public UsdMayaShaderWriter
```

Comment:
Same here. Missing MAYAUSD_CORE_PUBLIC tags for DLL export declaration.

Reply:
Should be addressed in https://github.com/Autodesk/maya-usd/pull/803/commits/6cd7599a2e4ef525f50766ad1ed161f84d62de2b.